src/codeu/chat/common/ServerInfo.java

```diff
@@ -26,4 +31,8 @@ public ServerInfo(Uuid version) {
   public String toString(){
     return "The Current Server Version is: "+ SERVER_VERSION;
   }
+  public ServerInfo(Time startTime) {
```

Comment:
move this constructor to the top with the other constructor (on top of the toString) and put the comment above the variable declaration